src/components/utils/file.js

```diff
@@ -23,13 +34,15 @@ const decodeBase64 = (image) => {
   return {integerArray, mimeString}
 }
 
-export const base64toFile = (image) => {
+const createBlob = (arr, type) => {
+  const blob = new Blob([arr], {type})
+  blob.objectUrl = URL.createObjectURL(blob)
```

Comment:
I have just been reading about this `createObjectURL`, it might not be the best solution. Since it's easy to create a memory leak with this. Your previous solution was actually better, since the ObjectURL got created at presentation time, but it still wasn't taking into account the case where the URL should be deallocated.